src/components/Person/Person.jsx

```diff
@@ -1 +1,48 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Person = ({
+  person: {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  },
+}) => {
+  const partnerGender = sex === 'm'
+    ? 'wife'
```

Comment:
would be better if you do like this 
```suggestion
export const Person = ({ person }) => {
  const {
    name,
    age,
    sex,
    isMarried,
    partnerName,
  } = person;
  
  const partnerGender = sex === 'm'
    ? 'wife'
```